src/network/NetworkController.ts

```diff
@@ -26,6 +28,8 @@ export enum NetworksChainId {
   ropsten = '3',
   localhost = '',
   rpc = '',
+  optimism = 'a',
```

Comment:
As far as I understood along the rest of the controllers we expect networks chain ids to be a base 10 numeric string (so this would be `'10'`). Can you confirm @Gudahtt @brad-decker?

Reply:
@wachunei during my testing the chain ids were base 16

Reply:
😨 Don't scare me lol, because kovan right on this enum is `'42'` and that's `66` if that was hex

Reply:
Tested and updated